department-of-veterans-affairs/caseflow
# Enable single-option new claimant relationship dropdown

Connects #14948 

### Description
As per dogfooding learnings, this PR updates the add claimant modal to be in line with our preference for an active single-option dropdown over a grayed-out dropdown.

### Acceptance Criteria
- [x] "Claimant's relationship to the Veteran" field is not grayed-out
- [x] The drop-down result defaults to "Attorney (previously or currently)"
- [x] A user can open the drop-down, but the only option they see is "Attorney (previously or currently)"

### Testing Plan
1. As a BVA Intake user, start an intake for a veteran.
2. Select Yes for "Is the claimant someone other than the Veteran?" and click "Add Claimant"
3. Verify all AC.

### User Facing Changes
Before this change:
<img width="462" src="https://user-images.githubusercontent.com/282869/90157161-5ff6e580-dd5b-11ea-955a-0ce0c19ef6cc.png">

After this change:
<img width="459" src="https://user-images.githubusercontent.com/282869/90157192-6dac6b00-dd5b-11ea-964d-8e4a9460568d.png">

Verifying that there is only one option (Claimant name menu is visible at bottom):
<img width="460" src="https://user-images.githubusercontent.com/282869/90157381-aa786200-dd5b-11ea-94f4-9f49bfa68108.png">
